src/core/database.py

```diff
@@ -0,0 +1,73 @@
+# coding: utf8
+
+__contact__ = "info@hytech-imaging.fr"
+__copyright__ = "Copyright (c) 2021 Hytech Imaging"
+
+import os.path
+
+from PyQt5.QtWidgets import QMessageBox
+from qgis._core import (
+    QgsWkbTypes,
+    QgsFields,
+    QgsVectorFileWriter,
+    QgsCoordinateReferenceSystem,
+    QgsCoordinateTransformContext,
+    QgsFeatureSink,
+    QgsVectorLayer,
+    QgsProject,
+)
+
+
+class SammoDataBase:
+    CONST_DB_NAME = "sammo-boat.gpkg"
+    CONST_LAYER_NAME = "session datas"
+
+    def __init__(self):
+        pass
+
+    def IsDataBaseAvailableInThisDirectory(self, directory):
+        return os.path.isfile(directory + "/" + self.CONST_DB_NAME)
```

Comment:
We should use `os.path.join(directory, CONST_DB_NAME)` for string concatenation.

Reply:
Done 6th commit (f9099ae08ce5c3734cf4631a5e052f67b1a83ea9)